code/public/index.html

```diff
@@ -4,31 +4,18 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
 
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>Technigo React App</title>
+    <meta property="og:title" content="Popular movies by Jenny Fiskaare & Linnea Frisk" />
+    <meta property="og:description" content="A site where you can scroll through the latest popular movies"/>
+    <meta property="og:image" content="https://i.postimg.cc/y8bBnsfR/Sk-rmklippmovie.png"/>
+    <meta property="og:url" content="https://project-movies-linnea-and-jenny.netlify.app/"/>
```

Comment:
Nice that you used this property to make content shareable! :) I tried it out by adding your netlify link to create a post on LinkedIn and the preview looks nice! :) Do you mind explaining the image, how did you create this? Is it basically a screen shot of the netlify page? I would like to implement this myself on future projects so thanks for the inspiration :)

Reply:
This is so interesting, learned something new!